### PR TITLE
Proposal 0: template

### PR DESCRIPTION
**Pull request by TizianoPerrucci**

### CODE REVIEW

File: proposals/0.template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Goal
+To drive the governance of the Squads community we want to record agreements and their review process.
+
+Those agreements are initiated by a community member (the driver) in form of a Proposal, and all other members can contribue to it.
+
+The Proposal will be up for debate until no objections are raised and a final agreement is reached. 
+
+## Proposal template
+- driver
```
Comment (iwein) [inline proposals/0.template.md:9]: I think there is going to be some confusion about what a Driver is (introduced last meeting). I would suggest that we stick with this definition for Driver: https://patterns.sociocracy30.org/driver.html (a person or group's motivation to respond to a situation). That definition is fundamentally different from the definition implied in this text.

That creates the need for a name for the person taking care of the proposal…  'maintainer' maybe?

Comment (TizianoPerrucci) [reply proposals/0.template.md:9]: sure we can rename driver into maintainer

Comment (iwein) [reply proposals/0.template.md]: adde proposed solution to this PR

**Review by iwein — APPROVED** [review]
lgtm

**Pull request merged**